client/me/purchases/payment-methods/existing-credit-card.tsx

```diff
@@ -0,0 +1,406 @@
+import { Gridicon } from '@automattic/components';
+import { Button, FormStatus, useLineItems, useFormStatus } from '@automattic/composite-checkout';
+import { PaymentLogo } from '@automattic/wpcom-checkout/src/payment-method-logos';
+import { SummaryLine, SummaryDetails } from '@automattic/wpcom-checkout/src/summary-details';
+import styled from '@emotion/styled';
+import { sprintf } from '@wordpress/i18n';
+import { useI18n } from '@wordpress/react-i18n';
+import debugFactory from 'debug';
+import { useTranslate } from 'i18n-calypso';
+import { Fragment, useCallback, useState } from 'react';
+import { useMutation, useQuery, useQueryClient } from 'react-query';
+import { useDispatch } from 'react-redux';
+import { getPaymentMethodSummary } from 'calypso/lib/checkout/payment-methods';
+import wpcom from 'calypso/lib/wp';
+import { errorNotice } from 'calypso/state/notices/actions';
+import PaymentMethodEditButton from './components/payment-method-edit-button';
+import PaymentMethodEditDialog from './components/payment-method-edit-dialog';
+import RenderEditFormFields from './components/payment-method-edit-form-fields';
+import type { PaymentMethod, ProcessPayment, LineItem } from '@automattic/composite-checkout';
+
+const debug = debugFactory( 'wpcom-checkout:existing-card-payment-method' );
```

Comment:
Minor, but we should probably change this back to `calypso:`
```suggestion
const debug = debugFactory( 'calypso:existing-card-payment-method' );
```

Reply:
Fixed.